### PR TITLE
chore: use github issue forms

### PR DESCRIPTION
**Pull request by DoctorJohn**
### Description

For some reason a lot of issues filled for this repo don't follow the issue templates. This PR updates our issue templates to use [Github issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms). They are currently in public beta and work quite well from my experience. They also come with a bunch of handy features which this PR makes use of:

- Issue fields can be marked as required and then cannot be left out by reporters
- Reporters interact with actual form elements (eg. checkboxes instead of handwritten [x])
- We can set input types, eg. `json` if we request the contents of `app.json`, the posted issues are then rendered nicely

I pushed those changes to [my forks main branch](https://github.com/DoctorJohn/react-native-google-ads), so the new issue templates can be viewed and tested there :)

### Checklist

- I read the [Contributor Guide](https://github.com/invertase/react-native-google-mobile-ads/blob/main/CONTRIBUTING.md)
  and followed the process outlined there for submitting PRs.
  - [x] Yes
- My change supports the following platforms;
  - [ ] `Android`
  - [ ] `iOS`
- My change includes tests;
  - [ ] `e2e` tests added or updated in `__tests__e2e__`
  - [ ] `jest` tests added or updated in `__tests__`
- [ ] I have updated TypeScript types that are affected by my change.
- This is a breaking change;
  - [ ] Yes
  - [x] No

### Test Plan

Head over to [my forks main branch](https://github.com/DoctorJohn/react-native-google-ads) and create a new issue to see the issue forms in action :)

:fire: 

---

Think `react-native-google-mobile-ads` is great? Please consider supporting the project with any of the below:

- 👉 Star this repo on GitHub ⭐️
- 👉 Follow [`Invertase`](https://twitter.com/invertaseio) on Twitter

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+title: "[\U0001F41B] Bug Report Title - CHANGE ME "
+labels: ["help wanted"]
+assignees: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Hello there you awesome person;**
+        Please note that the issue list of this repo is exclusively for bug reports;
+        If this is a setup issue then please make sure you've correctly followed the setup guides, most setup issues such as 'duplicate dex files', 'default app has not been initialized' etc are all down to an incorrect setup as the guides haven't been correctly followed.
```
Comment (mikehardy) [inline .github/ISSUE_TEMPLATE/Bug_report.yml:12]:
```suggestion
        If this is a setup issue then please make sure you've correctly followed the setup documentation, read [the common reasons for ads not showing](https://docs.page/invertase/react-native-google-mobile-ads/common-reasons-for-ads-not-showing), and searched the existing issues for your error message / issue. Most setup issues have been seen before and are all down to an incorrect setup as the guides haven't been correctly followed.
```

File: .github/ISSUE_TEMPLATE/Bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: |
+        Please describe your issue here and provide as much detail as you can.
```
Comment (mikehardy) [inline .github/ISSUE_TEMPLATE/Bug_report.yml:18]:
```suggestion
        Please describe your issue here and provide as much detail as you can.
        Stack Overflow publishes [a handy guide describing how to ask a question](https://stackoverflow.com/help/how-to-ask) in a way that improves your chances of getting useful help. Go there and read it quickly before filing an issue. Thank you!
```

**Review by mikehardy — COMMENTED** [review]
This should help a lot, issue quality is a thing and the correct approach is to try to improve initial input as opposed to correct after the fact, thank you

I made a couple of additions that I think may help - what do you think?

**Comment by DoctorJohn** [comment]
Your changes look sensible to me, thanks Mike! Also just checked that they render correctly in the form :+1:

**Review by mikehardy — APPROVED** [review]
Excellent - easy to iterate if we think of any further improvements - you interested in sharing maintainer permissions over here? That offer comes with zero obligation, you can do zero or more, always your option, but you have been a great help and I believe that (within reason) the more maintainers the better such that there is usually someone around with ability to make needed changes + reducing load on all at the same time

**Pull request merged**